migrate-base.js

```diff
@@ -1,99 +1,101 @@
-	const QuorumVote = await deploy(require('@daostack/arc/build/contracts/QuorumVote.json'));
-	const SimpleICO = await deploy(require('@daostack/arc/build/contracts/SimpleICO.json'));
-	const TokenCapGC = await deploy(require('@daostack/arc/build/contracts/TokenCapGC.json'));
-	const VestingScheme = await deploy(require('@daostack/arc/build/contracts/VestingScheme.json'));
-	const VoteInOrganizationScheme = await deploy(require('@daostack/arc/build/contracts/VoteInOrganizationScheme.json'));
-	const OrganizationRegister = await deploy(require('@daostack/arc/build/contracts/OrganizationRegister.json'));
-	const Redeemer = await deploy(
-		require('@daostack/arc/build/contracts/Redeemer.json'),
-		['ContributionReward', 'GenesisProtocol'],
-		ContributionReward,
-		GenesisProtocol
-	);
+  const DaoCreator = await deploy(
+    require('@daostack/arc/build/contracts/DaoCreator.json'),
+    ['ControllerCreator'],
+    ControllerCreator
+  );
+  const UController = await deploy(require('@daostack/arc/build/contracts/UController.json'));
+  const GenesisProtocol = await deploy(
+    require('@daostack/arc/build/contracts/GenesisProtocol.json'),
+    ['DAOToken'],
+    DAOToken
+  );
+  const SchemeRegistrar = await deploy(require('@daostack/arc/build/contracts/SchemeRegistrar.json'));
+  const UpgradeScheme = await deploy(require('@daostack/arc/build/contracts/UpgradeScheme.json'));
+  const GlobalConstraintRegistrar = await deploy(
+    require('@daostack/arc/build/contracts/GlobalConstraintRegistrar.json')
+  );
+  const ContributionReward = await deploy(require('@daostack/arc/build/contracts/ContributionReward.json'));
+  const AbsoluteVote = await deploy(require('@daostack/arc/build/contracts/AbsoluteVote.json'));
+  const QuorumVote = await deploy(require('@daostack/arc/build/contracts/QuorumVote.json'));
+  const SimpleICO = await deploy(require('@daostack/arc/build/contracts/SimpleICO.json'));
+  const TokenCapGC = await deploy(require('@daostack/arc/build/contracts/TokenCapGC.json'));
+  const VestingScheme = await deploy(require('@daostack/arc/build/contracts/VestingScheme.json'));
+  const VoteInOrganizationScheme = await deploy(require('@daostack/arc/build/contracts/VoteInOrganizationScheme.json'));
+  const OrganizationRegister = await deploy(require('@daostack/arc/build/contracts/OrganizationRegister.json'));
+  const Redeemer = await deploy(
+    require('@daostack/arc/build/contracts/Redeemer.json'),
+    ['ContributionReward', 'GenesisProtocol'],
+    ContributionReward,
+    GenesisProtocol
+  );
 
-	// deploy Reputation
-	await deploy(
-		require('@daostack/arc/build/contracts/Reputation.json'),
-		[]
-	);
+  // deploy Reputation
+  await deploy(
+    require('@daostack/arc/build/contracts/Reputation.json'),
+    []
+  );
 
-	return {
-		base: addresses,
-	};
+  const GenericScheme = await deploy(require('@daostack/arc/build/contracts/GenericScheme.json'));
```

Comment:
Does const GenericScheme is redundant ?

Reply:
yes, it is, but that is how the code is patterned and I didn't task myself here with refactoring the code

Reply:
But I went ahead and deleted all those anyway...